### ISSUE TITLE
Anchor and DocumentHandler connection

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#1247]
We need to add support for showing _specific_ DocumentHandler pages when using Anchor.

I.e.: user navigates to some part of DocumentHandler, enables some layers and create a share link using Anchor. The layers and zoom are restored when the link is opened, but DocumentHandler starts from the first overview page, rather than the one that was selected when the link was created.

**Comment by Hallbergs** [comment]
Very good idea! Could it be a good idea to include the actual chapter the user was at as well?

**Comment by jacobwod** [comment]
Yes, that's my intention (I understand it wasn't clear enough… 🙂 ).
